insights/specs/default.py

```diff
@@ -559,8 +559,7 @@ def block(broker):
     softnet_stat = simple_file("proc/net/softnet_stat")
     software_collections_list = simple_command('/usr/bin/scl --list')
     spfile_ora = glob_file("${ORACLE_HOME}/dbs/spfile*.ora")
-    ss = simple_command("/usr/sbin/ss -tulpn")
-    ss_tupna = simple_command("/usr/sbin/ss -tupna")
+    ss = first_of([simple_command("/usr/sbin/ss -tupna"), simple_command("/usr/sbin/ss -tulpn")])
```

Comment:
@vishwanathjadhav I don't believe that first_of is necessary here since both of the parsers can parse the `ss -tupna` command.  You can just have a single simple command.

Reply:
Done